integration_tests/build_test_containers.sh

```diff
@@ -1,14 +1,10 @@
 #!/usr/bin/env bash
 
-if [[ "$(docker images -q d3a-api-client 2> /dev/null)" == "" ]]; then
-  docker build -t d3a-api-client .
-fi
-
-if [[ "$(docker images -q d3a 2> /dev/null)" == "" ]]; then
+if [[ "$(docker images -q d3a-tests 2> /dev/null)" == "" ]]; then
   cd tests/ && \
-  git clone https://github.com/gridsingularity/d3a.git && \
+  git clone -b feature/D3ASIM-1922 https://github.com/gridsingularity/d3a.git && \
```

Comment:
please revert this before merging. Thanks!

Reply:
Delete or files